sdk/core/Azure.Core.Experimental/src/CloudMachine.cs

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+using System.IO;
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure
+{
+    /// <summary>
+    /// Stores configuration of a CloudMachine.
+    /// </summary>
+    public class CloudMachine
+    {
+        /// <summary>
+        /// Unique identifier of a CloudMachine. It's the name of the resource group of all the CloudMachine resources.
+        /// </summary>
+        public string Id { get; private set; }
```

Comment:
I don't believe that you need the private setter on these; it looks like you're only setting them from the constructors.